src/components/Checkbox/CheckboxStyled.js

```diff
@@ -58,6 +59,12 @@ export const ConsentDefinitionStyled = styled.div.attrs(props => ({
     css`
       opacity: 1;
     `}
+
+  ${isRTL() &&
+    css`
+      padding-left: 0;
```

Comment:
If you use logical properties instead of the cartesian top/right/bottom/left directions, then adding the `lang="rtl"` attribute will flip your styles automatically.
So instead of setting sth like

```
p {
  padding-left: 2em;
  padding-right: 5em;
}

[dir="rtl"] p {
  padding-left: 5em;
  padding-right: 2em;
}
```

you can just do
```
p {
  padding-inline-start: 2em;
  padding-inline-end: 5em;
}
```

or just
```
p {
  padding-inline: 2em 5em;
}
```

and setting `dir="rtl"` on the element takes care of the rest: https://codepen.io/kjhank/pen/XWxOXRr

Reply:
Ok. Good to know. I will try that one.